api/src/main/java/com/datastrato/gravitino/authorization/SecurableObject.java

```diff
@@ -43,13 +44,13 @@
 public interface SecurableObject {
 
   /**
-   * The parent securable object. If the securable object doesn't have parent, this method will
-   * return null.
+   * The parent full name of securable object. If the securable object doesn't have parent, this
+   * method will return null.
    *
-   * @return The parent securable object.
+   * @return The parent full name of securable object.
    */
   @Nullable
-  SecurableObject parent();
+  String parentFullName();
```

Comment:
Can we use `String parent()`, which seems simpler, WDYT?

Reply:
OK, maybe it's enough for us to tell user this is a full name string.